codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/testutil/TestHelpers.kt

```diff
@@ -67,7 +67,7 @@ fun testRustSettings(
 
 private const val SmithyVersion = "1.0"
 fun String.asSmithyModel(sourceLocation: String? = null, smithyVersion: String = SmithyVersion): Model {
-    val processed = letIf(!this.startsWith("\$version")) { "\$version: ${smithyVersion.dq()}\n$it" }
+    val processed = letIf(!this.trimStart().startsWith("\$version")) { "\$version: ${smithyVersion.dq()}\n$it" }
```

Comment:
Out of curiosity, was this needed because we removed some `trimIndent()` somewhere?

Reply:
It was needed because I added `$version` to a test model (that was implicitly versioned to 1 before) and since I did 
```
"""
$version 
```
it wasn't picking it up 😅 
So I decided to make it a bit more robust.